core/src/main/scala/com/typesafe/tools/mima/core/MemberInfo.scala

```diff
@@ -65,6 +65,9 @@ class MemberInfo(val owner: ClassInfo, val bytecodeName: String, override val fl
 
   var isDeprecated = false
 
+  // The full 'Signature' attribute, which includes generics
+  var signature = ""
```

Comment:
I realise the attribute is "Signature", but seeing as we have a "Sig" and we describe this one as "the full signature" should we call it `fullSignature`?

I'm also happy to follow the naming convention scalac or scalap use.

Reply:
what are the naming conventions scalac or scalap use?

Reply:
I don't know.

Reply:
looks like scalac, like the JVM spec, fairly consistently uses the name 'descriptor' for what is called 'sig' in mima (even though that is not particularly... descriptive). I'll update to use that.

Reply:
Thanks!